src/components/Dailog.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="main_container" >
+    <div class="container" v-for="issue in issues" :key="issue.id">
+      <v-alert
+        v-show="new Date(issue.dueDate).getDate() - new Date().getDate()  <= 3"
```

Comment:
Always make sure to write complex logics in the script. Please put this v-show logic in a method.